### PR TITLE
Update op-geth

### PR DESCRIPTION
**Pull request by ajsutton**
**Description**

Update to latest op-geth version
Update the update-op-geth script to account for removal of go.work

**Metadata**
- Part of https://linear.app/optimism/issue/CLI-3281/spike-work-out-differences-between-gettransactionbyhash-response-on

### CODE REVIEW

File: ops/scripts/update-op-geth.py [hunk 1]
```diff
@@ -11,34 +10,10 @@
 
 
 def main():
-	for project in ('op-service', 'op-node', 'op-proposer', 'op-batcher', 'op-bindings', 'op-chain-ops', 'op-e2e', 'op-wheel'):
+	for project in ('.', 'op-wheel'):
```
Comment (ajsutton) [inline ops/scripts/update-op-geth.py:13]: Note that `indexer/go.mod` also references op-geth but wasn't previously included in the list to update.  Should it be added here?

Comment (mslipper) [reply ops/scripts/update-op-geth.py:13]: Yes, we should add the indexer here.

**Review by mslipper — APPROVED** [review]
LGTM pending indexer fix.

File: ops/scripts/update-op-geth.py (hunk 1 above)
Comment (ajsutton) [reply ops/scripts/update-op-geth.py]: Done.

**Pull request merged**